packages/lit-dev-content/src/components/litdev-tutorial.ts

```diff
@@ -69,89 +86,252 @@ export class LitDevTutorial extends LitElement {
   @state()
   private _preloadedHtml?: {idx: number; promise: Promise<string>};
 
+  /**
+   * Tutorial manifest json file
+   */
+  private get _manifest(): TutorialManifest {
+    return this._manifestTask.value ?? {steps: [{title: ''}]};
+  }
+
   /**
    * Whether the tutorial is currently in its "solved" state.
    */
   private _solved = false;
 
+  private get _projectLocation() {
+    // TODO(e111077): delete this once we go public with tutorials catalog
+    if (window.location.pathname === '/tutorial/') {
+      return 'intro-to-lit';
+    }
+    return window.location.pathname.replace('/tutorials/', '');
```

Comment:
Minor nit: Test and remove trailing `/`.

Because the server adds trailing slashes, this currently behaves like so:

```
// While on the page https://pr655-606467e---lit-dev-5ftespv5na-uc.a.run.app/tutorials/intro-to-lit/
// tested in console:
window.location.pathname.replace('/tutorials/', '')
// Returns: 'intro-to-lit/'
```

This results in double slashes in requests, i.e.
https://pr655-606467e---lit-dev-5ftespv5na-uc.a.run.app/samples/tutorials/intro-to-lit//tutorial.json